R/dbSendQuery.R

```diff
@@ -28,11 +28,15 @@ NULL
 .dbSendQuery <- function(conn, statement, ...) {
   url <- paste0(conn@host, ':', conn@port, '/v1/statement')
   status <- 503
+  retries <- 3
   headers <- .request.headers(conn)
-  while (status == 503) {
+  while (status == 503 || (retries > 0 && status >= 400)) {
```

Comment:
I can use `httr::http_error` (which actually checks for `>= 400` but I'm sure would quickly be updated if things change :))?